### PR TITLE
chore: run e2e/js_run_devserver under bzlmod

### PR DESCRIPTION
**Pull request by kormide**
In support of https://github.com/aspect-build/rules_js/issues/644.

### CODE REVIEW

File: e2e/js_run_devserver/WORKSPACE [hunk 1]
```diff
@@ -35,18 +35,6 @@ npm_repositories()
...
-    sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",
-    urls = ["https://github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip"],
-)
-
-load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
-
-go_rules_dependencies()
-
-go_register_toolchains(version = "1.19.3")
-
```
Comment (kormide) [inline e2e/js_run_devserver/WORKSPACE:49]: Go wasn't used in this e2e.

File: e2e/js_run_devserver/MODULE.bazel [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        "http-server": [
+            "http-server=./bin/http-server",
+        ],
+    },
+    npmrc = "//:.npmrc",
+    pnpm_lock = "//:pnpm-lock.yaml",
+    verify_node_modules_ignored = "//:.bazelignore",
+)
+
+use_repo(npm, "npm")
```
Comment (alexeagle) [inline e2e/js_run_devserver/MODULE.bazel:32]: Nit, could import both in one use_repo call, like one load statement

File: e2e/js_run_devserver/WORKSPACE.bzlmod [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# The presence of this file causes WORKSPACE to be ignored when bzlmod is enabled.
+# See https://docs.google.com/document/d/1JtXIVnXyFZ4bmbiBCr5gsTH4-opZAFf5DMMb-54kES0/edit#heading=h.y054fjub9max
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "com_github_ash2k_bazel_tools",
```
Comment (alexeagle) [inline e2e/js_run_devserver/WORKSPACE.bzlmod:7]: Let's file these on Bazel-central-registry as we encounter them. I'd like to ask Google to fund bzlmod rollout but we have to show them a "percent missing" or something

**Review by alexeagle — APPROVED** [review]

File: e2e/js_run_devserver/WORKSPACE.bzlmod (hunk 3 above)
Comment (kormide) [reply e2e/js_run_devserver/WORKSPACE.bzlmod:7]: https://github.com/bazelbuild/bazel-central-registry/issues/419

**Pull request merged**